dotnet/src/SemanticKernel.Core/Text/TextChunker.cs

```diff
@@ -240,12 +237,17 @@ private static (List<string>, bool) Split(List<string> input, int maxTokens, Rea
         return (result, inputWasSplit);
     }
 
-    private static (List<string>, bool) Split(ReadOnlySpan<char> input, string? inputString, int maxTokens, ReadOnlySpan<char> separators, bool trim, TokenCounter tokenCounter)
+    private static (List<string>, bool) Split(ReadOnlySpan<char> input, string? inputString, int maxTokens, ReadOnlySpan<char> separators, bool trim, TokenCounter? tokenCounter)
```

Comment:
Is it possible to use a named tuple for better readability
```suggestion
    private static (List<string> splitResult, bool hasSplit) Split(ReadOnlySpan<char> input, string? inputString, int maxTokens, ReadOnlySpan<char> separators, bool trim, TokenCounter? tokenCounter)
```

Reply:
It could use names.  Wasn't changing that here, though.